plugins/Homologues.py

```diff
@@ -50,10 +52,20 @@ def download_species(uri, release, staging, download, species: str) -> ManifestR
         # Create the resource info.
         resource_stage = Dict()
         resource_stage.uri = protein_uri
-        resource_stage.output_filename = f'{release}-{species}.json'
+        resource_stage.output_filename = f'{species}.json'
         resource_stage.output_dir = staging
         return download.ftp_download(resource_stage)
 
+    @staticmethod
+    def download_homologies_for_species(uri_homologues, release, output, type, species: str, download) -> ManifestResource:
+        # TODO - create download logic
+        protein_uri = f"{uri_homologues}{species}/Compara.{release}.{type}_default.homologies.tsv.gz"
```

Comment:
This part is about downloading the homology data, we should rename the variable 'protein_uri' to 'download_uri', as this is going to iterate through all the homology file types set in the configuration file.